bff/test/Duende.Bff.Tests/Endpoints/Management/UserEndpointTests.cs

```diff
@@ -79,7 +79,7 @@ public async Task when_configured_user_endpoint_for_unauthenticated_user_should_
             BffHost.BffOptions.AnonymousSessionResponse = AnonymousSessionResponse.Response200;
 
             var data = await BffHost.CallUserEndpointAsync();
-            data.ShouldBeNull();
+            data.ShouldBeEmpty();
```

Comment:
We should rename the test "... _and_empty"

Reply:
fixed